client/blocks/reader-avatar/style.scss

```
@@ -30,6 +30,12 @@
 			min-width: 32px;
 			min-height: 32px;
 		}
+		&.has-gravatar.has-site-icon {
+			max-height: 40px;
+			max-width: 40px;
+			min-height: 40px;
+			min-width: 40px;
```

```suggestion
			max-height: 40px;
			max-width: 40px;
```

Looks like these rules are duplicated :)